system-tests/_support/index.js

```diff
@@ -219,45 +219,56 @@ Cypress.addChildCommand("triggerHover", function(elements) {
 });
 
 /**
- * Launches a new service in group TEST_UUID using the dcos CLI
+ * Launches a new service using the dcos CLI. Service must be created
+ * in the TEST_UUID group.
  *
  * @param {Object} serviceDefinition - The service JSON definition file
  *
  */
 export function createService(serviceDefinition) {
+  const idParts = serviceDefinition.id.split("/");
+  if (idParts.length !== 3) {
+    throw new Error("Must include leading slash in service id");
```

Comment:
@ahoskins (Nit): What do you think about doing `!id.startsWith("/")` for better readability?